storage/allocator.go

```diff
@@ -41,6 +42,10 @@ const (
 	// is within rebalanceFromMean of the mean, it is considered a
 	// viable target to rebalance to.
 	rebalanceFromMean = 0.025 // 2.5%
+	// rebalanceShouldRebalanceChance represents a chance that an individual
```

Comment:
This comment needs a unit of time for the chance to make sense. "per ScanInterval"?  5% random movement every 10 minutes strikes me as insanely high, but we'll get a better sense of what's reasonable once we have Bram's simulator.